### DISCUSSION TITLE
zh/guide/drivers/lanzou

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#115] [Announcements]
# zh/guide/drivers/lanzou

兰奏网盘挂载有两种模式（类型），分别的 cookie 和 链接 cookie：可以显示网盘所有文件，但是需要提供网盘的cookie; 链接：只能显示链接里面的内容（不需要提供cookie就可以挂载）; ⚠推荐使用cookie的方式挂载，用链接模式总是会抽风不知道怎么回事突然提示：failed get objs:failed to list objs: ...

https://alist.nn.ci/zh/guide/drivers/lanzou.html

**Comment by izelle** [comment]
挂在成功后，根本不能用啊，文件夹里的文件被显示成“请忽使用第三方工具”

**Reply by jeva13** [reply]
我也是一毛一样的，挂载后文件直接显示“请勿使用第三方工具”

**Reply by kaiser20106** [reply]
一样

**Comment by gy-0** [comment]
“蓝奏”打成“兰奏”了

**Comment by joezhn2020** [comment]
之前还能下载不能上传，现在啥都做不了“请忽使用第三方工具”

**Comment by ChaXxl** [comment]
使用 cookie 登录，设置完cookie 之后，failed get objs: failed to list objs: login not

**Reply by lingyu0628** [reply]
+1

**Comment by fusheng120** [comment]
使用链接分享的时候发现cookie是必选项😂

**Comment by zzerosama** [comment]
群晖用cloudsync上传123网盘时，显示上传失败，文件已由远程服务器锁定是怎么回事儿

**Comment by Harrison3-1122** [comment]
分享链接有解决方案吗

**Comment by ai1239** [comment]
蓝奏出了桌面，禁止用第三方没办法

**Comment by lonely-4** [comment]
6,这用不了😂😂😂😂🤣🤣🤣😅😅😅

**Reply by fyr611** [reply]
蓝秦云无法下载exe文件了。压缩包文件不受影响。

**Comment by zhoupingxiao** [comment]
提示：{"code":500,"message":"failed link: failed get link: 已超时，请刷新","data":null}

**Reply by teyonds** [reply]
你找到原因没

**Reply by zhoupingxiao** [reply]
没有，之前可以用，突然不行的。

**Reply by shubaofx** [reply]
还没解决么

**Comment by teyonds** [comment]
failed link: failed get link: 已超时，请刷新

提示这个咋整，账号和Cookie方式都不行

**Reply by lonely-4** [reply]
我也是

**Comment by shubaofx** [comment]
文件有了 点下载出现这个
{"code":500,"message":"failed link: failed get link: not find data","data":null}

**Reply by lonely-4** [reply]
同感...

**Comment by teyonds** [comment]
还没推更新，测试里面更新了



在 2023-08-29 17:19:47，"山鬼网络科技" ***@***.***> 写道：

还没解决么

**Comment by clearng** [comment]
用账号密码的方式登录，能正常上传，但是不能打开和下载文件，提示`failed link: failed get link: not find file page param`
![img](https://img.keliyan.top/file/40dad61d2fbeb4e39a123.png)

**Reply by 1772891600** [reply]
我也是这样

**Comment by goldsoft8888** [comment]
限制文件大小就算了，还限制文件名，限制文件名也就算了，还审查文件内容格式，上传时，提示无法识别文件格式，这还怎么用？
还不让换名,要会员才行，你们是不想让别人用吧？不敢购买了

**Comment by goldsoft8888** [comment]
”无法识别文件内容，请联系客服处理。“    你还想看我文件内容啊？    正常时上传1.7M字节，不正常时，官方网站都打不开，差评，是我见过的最恶心的网盘

**Comment by ghost** [comment]
我可以用，但上传不了文件（2023.11.26）

**Comment by 024k** [comment]
# 我现在用的AList版本是v3.23.0，添加蓝奏云盘，没有账户模式了………………

**Reply by lava191810** [reply]
我用的3.26.0，蓝奏云盘有账户模式啊

**Comment by lava191810** [comment]
用的cookie登录，除了zip点其他的都是failed link: failed get link: not find file sgin

**Reply by lava191810** [reply]
发重了
你们当没看见

**Comment by lava191810** [comment]
failed link: failed get link: not find file sgin
除了zip之外的文件点了都是这个报错
上传和目录显示正常
使用cookie登录

**Reply by lava191810** [reply]
更新了一下AList，现在正常了

**Reply by czjdr** [reply]
@lava191810 你好，请问你能正常上传下载吗？我下载报错500

**Reply by lava191810** [reply]
可以啊

**Comment by Masterain98** [comment]
有没有朋友遇上上传文件产生524错误的问题？无论是否添加为任务，Stream和Form都试过了，文件在大于50M之后就一定会出现524错误。使用蓝奏云官方的后台上传是没有问题的，使用alist上传更小的文件是也是可以成功的（测试过最大30M的文件是没问题的）。

前台传输显示错误为： Request failed with status code 524
后台任务显示错误为：Post "https://up.woozooo.com/html5up.php": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Comment by wayner6** [comment]
无法下载，提示
{"code":500,"message":"failed link: failed get link: not find file page param","data":null}

**Reply by 1772891600** [reply]
是的，我也是

**Comment by purainity** [comment]
使用链接挂载无法获取文件列表，提示
failed get objs: failed to list objs: not find data

**Comment by djwlx** [comment]
如果出现{"code":500,"message":"failed link: failed get link: not find file page param","data":null} 的时候，可以去蓝奏云官网个人中心查看一下个性域名，然后把分享链接改成个人域名就可以了

**Reply by 1772891600** [reply]
也不行

**Comment by purainity** [comment]
> 使用链接挂载无法获取文件列表，提示 failed get objs: failed to list objs: not find data

这种现象已解决，是 AList 自带的蓝奏云分享链接有误，改成 `https://wwop.lanzoul.com` 就好了

![Screenshot_20240210111135](https://github.com/alist-org/docs/assets/126877770/4636993d-cecb-472d-8135-2c2801b5607c)

**Comment by lumengde123** [comment]
对，去蓝奏云的个人中心->个性域名，复制到设置里的分享链接里。刷新下主页的缓存或者设置里的缓存暂时设为0分钟。再打开就能下载了！（还有个原因，有的文件被蓝奏云官方删了，但是目录还在，如果Alist去下载也报错，需要自己在蓝奏云里下载下看看行不，不存在的文件手动删了）

**Comment by zhangzong666** [comment]
failed link: failed get link: not find file page param
我什么方法都用了，官网还是可以用，alist咋样都用不了，我把分享链接也换了，还是这样，望修复，这种情况有很多！麻烦您了！

**Reply by caishen9423** [reply]
我也是这样。。创建索引没问题，但是点进去，就弹这个了

**Comment by 1772891600** [comment]
显示：failed link: failed get link: not find file page param

**Comment by OORTaka0825** [comment]
蓝奏云下载显示{"code":500,"message":"failed link: failed get link: not find file page param","data":null}，什么换个性域名什么改分享链接都试了还是不行，有没有解决的大哥@我一下

**Reply by shiquda** [reply]
我换成`https://wws.lanzoux.com`就可以了

**Comment by caishen9423** [comment]
这个是UA问题，大佬已经解决完了，但是还没打包。。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/docs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月4日(星期天) 凌晨1:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/docs] zh/guide/drivers/lanzou (Discussion #115)





 
蓝奏云下载显示{"code":500,"message":"failed link: failed get link: not find file page param","data":null}，什么换个性域名什么改分享链接都试了还是不行，有没有解决的大哥@我一下

**Comment by kyring2701** [comment]
Post "https://pc.woozooo.com/html5up.php": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

这咋办

**Comment by leonvogle** [comment]
failed get objs: failed to list objs:

**Reply by leonvogle** [reply]
使用的账号模式，但是不行，报错。

**Comment by panluoyant** [comment]
目前账户密码没问题了

**Reply by leonvogle** [reply]
还是提示failed get objs: failed to list objs:
